changelog_entry.yaml

```
@@ -0,0 +1,4 @@
+- bump: minor

```

```suggestion
- bump: patch
```
Since it doesn't introduce new functionality